README.md

```diff
@@ -3,8 +3,9 @@
 ![python testing](https://github.com/SimpleITK/CommentSpellCheck/actions/workflows/python-app.yml/badge.svg)
 
 
-A script that automatically spell checks the comments of a code base.
-It is intended to be run on the SimpleITK and ITK code bases.
+The CommentSpellCheck (CSC) package provides a script that automatically
+spell checks the comments of a code base.  It is intended to be run on
```

Comment:
I would not limit this by saying "it is intended to be run", maybe say "it was originally developed to be run on..."